doc/source/reference.txt

```diff
@@ -3,7 +3,7 @@ DO NOT EDIT THIS LINE. This file is used in `conf.py to generate schema.md`. Edi
 .. _helm-chart-reference:
 ```
 
-# Configuration Reference
+# Helm Chart Reference
```

Comment:
IMO 'configuration reference' is the right thing - or 'Helm Chart Configuration Reference'? Helm Chart reference to me sounds like a reference for creating, maintaining and modifying helm charts. In this case, this is only reference for various configuration options for one particular helm chart (JupyterHub)

Reply:
Maybe: Configuration Reference for JupyterHub Helm Chart 

This is one of those places where explicit may be better than implicit ;-)

Reply:
+1

Reply:
yeah I agree - I changed it to `helm chart reference` originally because technically the entire website is a configuration reference :-) I think @willingc 's edits are the way to go